src/main/java/mil/dds/anet/beans/Report.java

```diff
@@ -265,35 +267,55 @@ public void setCancelledReason(ReportCancelledReason cancelledReason) {
     this.cancelledReason = cancelledReason;
   }
 
+  @GraphQLQuery(name = "reportPeople")
+  public CompletableFuture<List<ReportPerson>> loadReportPeople(
+      @GraphQLRootContext Map<String, Object> context) {
+    if (reportPeople != null) {
+      return CompletableFuture.completedFuture(reportPeople);
+    }
+    return AnetObjectEngine.getInstance().getReportDao().getPeopleForReport(context, uuid)
+        .thenApply(o -> {
+          reportPeople = o;
+          return o;
+        });
+  }
+
+  public List<ReportPerson> getReportPeople() {
+    return reportPeople;
+  }
+
+  @GraphQLInputField(name = "reportPeople")
+  public void setReportPeople(List<ReportPerson> reportPeople) {
+    this.reportPeople = reportPeople;
+  }
+
   @GraphQLQuery(name = "attendees")
   public CompletableFuture<List<ReportPerson>> loadAttendees(
       @GraphQLRootContext Map<String, Object> context) {
     if (attendees != null) {
       return CompletableFuture.completedFuture(attendees);
     }
-    return AnetObjectEngine.getInstance().getReportDao().getAttendeesForReport(context, uuid)
-        .thenApply(o -> {
+    return loadReportPeople(context) // Force the load of reportPeople
+        .thenApply(l -> {
+          final List<ReportPerson> o =
+              l.stream().filter(p -> p.isAttendee()).collect(Collectors.toList());
```

Comment:
lambda can be replaced with method reference i.e `filter(ReportPerson::isAuthor)` as it is easier to read.